src/style.css

```
@@ -0,0 +1,89 @@
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
```

```suggestion
@font-face {
  font-family: "Avenir";
  src: url(/fonts/Avenir-Book.ttf),
    url(/fonts/Avenir-Heavy.ttf),
    url(/fonts/Avenir.ttc);
  font-weight: 400;
}
```
you can move font declaration to separate file and add it to your html, than you won't have to add `font-family: "Avenir", sans-serif;` to every selector